controllers/dspipeline_params.go

```diff
@@ -610,6 +610,38 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if p.APIServer.CustomKfpLauncherConfig != nil {
+			if p.APIServer.CustomKfpLauncherConfig.Name != "" {
```

Comment:
nit:
```go
if p.APIServer.CustomKfpLauncherConfig != nil && p.APIServer.CustomKfpLauncherConfig.Name != "" {
```

Reply:
+1, you can remote the nesting as Helber suggested, and also store the ConfigMap name in a variable for readability and consistency. Something like:

```
configMapName := config.CustomKfpLauncherConfigMapName

if cfg := p.APIServer.CustomKfpLauncherConfig; cfg != nil && cfg.Name != "" {
    if cm, err := util.GetConfigMap(ctx, cfg.Name, p.Namespace, client); err != nil {
        if !apierrs.IsNotFound(err) {
            log.Info(fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", cfg.Name, err))
            return err
        }
    } else {
        p.APIServer.CustomKfpLauncherConfig = &dspa.KfpLauncherConfigMap{
            Name:                configMapName,
            <remaining-vars>
        }
    }
}

```

Reply:
Thanks for the suggestions @DharmitD and @hbelmiro. Modified the code for better readability now.